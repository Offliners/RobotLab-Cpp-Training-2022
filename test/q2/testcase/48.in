815
diMeNsioN
DrivE
DeVICE
etHiC
aging
PrIvILEGE
sin
THought
WAx
COST
CLOSE
ACCoMPany
DELAY
pink
ouT
gIfted
MOVABLE
CAKE
CHILDLIKE
03RCLE
TOfU
SORRy
WHOLLY
Dr0Er
PASSEnGER
DIArY
MAyor
MEChAnicAL
eNroll
1830R6
rEALitY
DelIBERaTEly
DiZzY
thousAND
army
DiffeR
taCKle
Pride
crAsH
PRincess
realistic
SAcK
comRade
HypOtheSiS
huMoR
uNDeSIRABLE
sIMplIFy
2rag5n1l3
PESSIMISM
CoNCRete
DevAstatIng
nonethelEss
vacancy
prInciPle
caBBA86
THuNdErsTORm
HAlT
MOaN
FAith
sea3
REVISiOn
VIAble
INHERIT
7L915e
Miserable
dra9atic
COMPANIonSHIP
ghosT
SPILL
divERsIOn
coOPerAtIve
FOLK
sOMEWhAt
22447y
tranquIlizeR
siTUateD
TReMBlE
accoUNt
C3AT
EAGER
p4geo9
STATE
outside
BAM
H6396Nd
E7186137e
fOrEignER
achieve
rOck
HypOcRIte
PiLot
DARKrOOM
goAT
jump
hObby
how
KNOW-HOW
date
GRaMMAr
crEatiOn
PALm
CULtURAL
SubJeCTIVE
DEL1NQUENT
001l
PRINTER
KNOt
contiNue
CorN
mess
furtHErmore
conTract
pAVEmENT
PeNIs
haRASSmENT
HaIL
MAtteR
STEAL
few
WeathER
frOg
DeLinquent
reSt
TRANSmISSION
mINUs
kItTEn
OBJeCTIVE
CHEF
INTERMEDIAtE
NOD
2r5vi8Eg1
PETROLeUm
Dinner
midnight
22629E76T216
SWEeP
cONTAINER
4AR8
N9UG274
c18Z1
SLIGHT
HYMN
homework
comiNg
PrEsIdentiAL
JAR
bUd
DaMages
sENsITiVE
inspIre
FORWArD
routine
AUGUST
DryER
ENABLE
brOAdcAsT
du3l
account
BLAN2eT
InVESTIGAtIoN
HArm13Y
ComMisSION
FOrWARD
cONVINcING
36677T
LOcomotive
CHANCE
oPTICaL
cuSTODy
JOG
rEvIEW
54851y8581
cabinet
COMMunITy
bookstoRE
humid
gENIuS
PoSSiBILity
BEHAVIoR
locaTioN
UsheR
AMAzE
TuRn
if
cODE
afford
aLLOCAte
NOMINEE
SUPERIORITY
Fate
disastrous
E-MAiL
couNcilMaN
honEymoon
sPLaSh
CArpEnTeR
distRACt
CUsTOMeR
LaNDlORD
ANECDOTE
REPRESENT
shepherd
PENNY
COffEE
DEFeCT
TIGHTLY
whoever
GlassWare
be6Ut6721
s176w1
eighteen
juice
dose
VAcUum
tEMporaRY
PIT
geNeRaLIzE
TOTaLlY
loudspeakeR
exceptiON
COUNTER
NeCEssArY
cIVILiAN
aUToNOmy
margin
TRAgEDY
aSiDe
HOlD
somEHow
INteLlECT
empEror
CONCEITEDLY
sIGNAL
reNAiSSance
GREEDY
CoNtEXT
U
SuperFLUous
DisCOVer
91145l
PHILOSoPhEr
eNOrmoUs
ANXiEtY
poWEr
dISABILITY
Claw
toRtoiSE
GAP
lunch
TU5S1A1
MOtorcycLe
abruPt
HAUL
i
cAmpuS
cArD
REpuBLIC
G4316ry
cHarIty
26f05enz8
beauTiful
croWDED
ISoLATE
SCENE
liaBiLiTy
various
weekly
reCOGNIze
ThESIs
baRBeCUe
516A1K
curfew
namE
gamble
8D2pTAT193
periODiCAL
information
people
OVERTuRn
o
merE
coMforTinG
affection
adaptaTion
TRAvERSE
8EPARATELY
GEAR
catcher
OrgaNIzatIOn
arouse
junior
saNITAtION
tOmOR752
shampoo
PACT
IMpRESS
vENUS
7NVENT1R
goodness
rAPE
63N203229410
MISHAP
gEoMeTRY
career
wAkE
biAs
relisH
belONGINGS
LeakagE
MANDARIN
ESCalATOr
CopiLOt
POlIce
C1900s5
reSisTAnce
8e5m
CoMpeTe
sNeaKEr
HanDLe
ROTATION
COMMERCE
oath
HElmET
INFLUENTIAL
cHain
INFLICt
LIVELIHOOD
outBreAk
NORtHERN
redUcE
BizaRRE
RED
engiNeer
9168N7E
song
MAP
ReplAceable
ExhibiT
simplicity
S9575
BOuNd
pu1zl8
nearsighted
TOWEr
proudLy
bri2h7
BUlLEt
cONTINuOUs
MUsiCIaN
dRaUght
COORDINATION
aNnual
DEPRESSION
UNDerGRADUaTE
manner
MIND
video
hurl
tENSE
28l285O22
INdIFFEREnt
POET
0537o9
HUMAN
fortunate
235w
cerTificaTE
rAid
modem
IMPaTIeNT
SpinAcH
7090U33
i31e2uiTy
DecEMBer
harmOny
a0pLica2T
RUin
pAW
THERaPIST
JaCKEt
cease
teEnAger
SURrOunDIngS
HuNGEr
AveNgE
PePpEr
SpELLInG
guArd
rESUME
RECOnSTRuCtIOn
HUMBLy
arTiLLerY
HEad-On
ASsISTANCE
suit
hOPe
FINANCIAL
INTRoDuce
ANGULAR
buSh
GUaRantEe
aBstRacTiOn
sEduce
nest
CLAY
LiStEn
CHILDHOOD
THRilLiNG
l200R4t4o4
CERTaIN
MErCHaNt
ALliAnce
sANctiOn
sTarvE
HoNOR
BAThroom
HARE
CHESSBOARD
go
amEnd
eNcHanT
HorRoR
DeSignAte
NeRvE
cOMMITMENT
OvERLAp
vISiBLE
iLLUMINATe
paTieNt
doom
JUVeNILE
6A25
CIRCULATION
ointment
S8ECIAL
GIRL
INTIMATE
CONSCIeNTIOUS
GrApE
SiMULTAneoUsLY
Each
ObScURE
lAUgh
94R3oN6
lOgic
gOWN
oriGInaliTy
5uS
DIVERSITY
mileage
SHOCK
weT
wear
760R
meaninglesS
COME
gRUmBLE
wHy
fEET
MOUNTAIN
aSsis1a7t
evokE
glORIOUS
INteLLIGeNT
EMOTIoNAL
SEntenCE
TENT
SIGNIFICANCE
relativE
SeIZE
3UR1
cOnTiNuAtIon
MonstrouS
deSert
INCLUDE
CLoSuRE
MediCatiOn
PLAyGROUND
6U27IcA95
DIgITaL
ACTIVITy
carrIeR
unIoN
herd
ARTIFICIAl
ENDURANCE
ordinaNce
MotTo
RELAX
cHeerLEAder
terminate
90P83585973
SNEEzE
ResULTaNT
aLlEY
sympaThetic
miCROpHonE
suBSTAnCe
MOBILE
B1n3AGE
bully
UrgeNCy
COMPETITOR
appReciaTion
INDULGE
bark
seAm
meTer
8I9
Recover
HesiTaNtLy
tuition
signature
BIKInI
ShArE
aDMIRAtION
breeDeR
fREsHMAN
crIsIS
BACkGROUND
917i2U6
ENgINEeRING
ACADEMIc
1e5orT
Probe
STORmY
defeNd
i
miSsiONary
teAR
WalL
stocKing
uLtravIolet
58am5
CAnDiDATE
sHEar
GrASSHopPer
GEmINI
CHEERFUL
trumPet
TAbLESPOOn
SICK
WeDNEsdAY
conceited
mystery
chip
MerrY
DeVOtEd
coNVEnIEncE
4A6338
52co7
lAnDslide
CabLE
SHUT
Co6PLe
ANn8AL
iNVestigATOr
dISQUALifY
AFTERMaTh
STATE
DULL
GUIlTY
prEsCRIBe
a6O2e
religion
philosOPhY
scRAP
BLACKBOARD
nuMeriCal
skinny
REspeCtIVe
IMPRACTICAL
cONtrOLLeR
7676r7
DISMAy
tAPE
TOol
fierce
diehard
naTIOn
indecision
CAlEndAr
sensit6V9
uNpREDiCtable
GROWTH
injury
beer
EYeSight
UnUSUaLLY
coNtEnT
InterchangeABle
conSisTent
BLEED
LIKING
claSsifICatiOn
distRIct
Bucket
legend
lOTiON
71p
nOMiNAtION
coMPartmENt
imPaCT
coMpetiTion
Commu6iTy
CoMfortaBly
SWallOW
pAlaCe
CoVEr
4End
total
DESCRiPTioN
ja5WAlk
sURPrISInG
sufficient
neat
meaNwhile
sE6EsteR
CORPORATION
DrIVEwaY
meaning
acTIon
homelEss
POPULATE
quaLIty
WHEAT
cONfesSiOn
latteR
MEDIUM
COMMEMORATE
cAuTIoUSlY
geOGraPHICal
DEMONSTraTE
dEmoCRACY
daYlight
LaD
DISSIDENT
LaMENT
Repay
lorD
naRrow
SuRGERY
educational
SegREGATE
GraiN
resTorE
0167TU74IO6
jUly
REPuBLIcaN
couragEous
safE
NEWsCaSTER
gENEROSITY
AqUARIuM
dance
rE74R1
convenient
SCOUT
huG
SNOWY
drowsy
CupBOArD
RIDe
cOnSpIracY
MuSICAL
DEDIcATE
PEDEstRIAN
discussion
ExAMINee
DATA
cUmulatiVe
IdeNtItY
mediCine
PERFUME
6Os17gE
hurRicane
sataN
prOmINent
c098U0
BlUe
jOUrNAlISM
sUBmiT
dEnse
cUtE
SomEONE
PrEFERAblE
MISCHIeF
Baby
oVerTHROw
CoMpUteRize
shEEP
MEAL
pizza
InNOCEnT
encyCLopEdIa
pUPPY
INVESTOR
REDuNDaNT
rAY
sAUCe
TeAm
VOice
7ayMA5
2R413R80l6
c23j33c7Ion
r67
gyMnaSIUM/GyM
cheErs
beijing
item
pOnD
WELFARE
bROchURe
VIOLENCE
2e2e8bl6
H86s7W088
BeYond
CArT
3a75i2
b5d57t
assoCIatiOn
mothErhood
ace
overwoRk
SPEED
FA31U7
mUSCLE
ACcusAtION
05409E88
MISBEHAVE
tHOrOuGh
cALCUlATiON
proDuCtIvIty
InTEgRAl
feel
Plow
bEnEfICiaL
tHerE
OccaSIonal
6ise
BAMBoo
mAGnifIceNt
manpower
iLlustRation
DolLaR
provinciAl
ConFlICT
coLleCTOr
horrible
meMber
alTERnATE
WHO
bag
METAL
similar
PRONouN
pUrSE
geneRouSly
flOck
Assault
fIeLd
oUTInG
Blow
director
DILUTE
DEFicIenCy
COmmUnicatE
precEDEnT
CARgo
err
ENVISAGE
HERE
truNK
booM
qUiTE
humOrOUsly
traditional
number
laboratory/lAb
apPEal
ExerciSe
cOuntrYsIDE
HARMFUL
3UNcT91n69
SPEAKer
REvolUTIon
booKCasE
EXTrEmE
C995
HELICOPTER
PILLOW
halftvay
BEHAVe
GENERATOr
APpoiN4
DiverSe
obJeCT
HaLf
PLa25
sMELl
pAl
cale11a1
airpL4ne
eLECtRIcIty
REFRESH
84E0Ag7
CoSmic
GLAD
u6gE70
pAnel
BOND
Clock
careleSs
EnclosE
gRam
