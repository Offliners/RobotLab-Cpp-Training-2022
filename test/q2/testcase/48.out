n. 尺寸；範圍
n./v. 車程；駕駛
n. 裝置
n. 倫理
adj. 變老的
n. 特權
n./v. 罪惡
n. 思想
n. 蠟
n./v. 花費；付出
adj./adv. 接近的
v. 陪同
n./v. 拖延
adj./n. 粉紅色的
prep/adv. 在外
adj. 有天賦的
adj. 可動的
n. 蛋糕
adj. 天真無邪的
Unknown
n. 豆腐
adj. 感到抱歉的；感到遺憾的
adv. 完全地
Unknown
n. 乘客
n. 日記
n. 市長；鎮長
adj. 機械的
v. 使加入；註冊
Unknown
n. 真實性
adv. 故意地
adj. 頭暈目眩的
adj./n. 一千
n. 軍隊；團體
v. 不同於…
v. 處理
n./v. 自豪
n./v. 墜毀；撞擊
n. 公主
adj. 理論的
n. 大袋子
n. 同伴
n. 假設
n. 幽默感
adj. 令人不快的
v. 使單純
Unknown
n. 悲觀(主義)
adj./n. 具體的；混凝土
adj. 毀滅性的
adv./conj. 但是；仍然
n. 空位
n. 原則
Unknown
n. 雷雨
v./n. 停止；暫停
v./n. 呻吟(聲)
n. 信念；諾言
Unknown
n. 修正
adj. 切實可行的
v. 繼承
Unknown
adj. 不幸的
Unknown
n. 交誼
n. 鬼；幽靈
n./v. 溢出；濺出
n.  轉換
adj. 合作的
n. (某一民族或社會階層中的)廣大成員
adv. 有點
Unknown
n. 鎮靜劑
adj. 坐落在…的
n./v. 發抖
n. 帳戶；描述
Unknown
adj. 熱切的
Unknown
n. 狀態
prep./adj./adv./n.  在外面；外部的
n. 穀倉；牲口棚
Unknown
Unknown
n. 外國人
v. (n)完成；達到
n. 岩石
n. 偽君子
n./v. 飛行員；駕駛
n. 暗房
n. 山羊
n./v. 跳躍
n. 嗜好
adv. 如何；多少
n. 技能
n. 棗；棗椰樹
n. 文法
n. 創作；萬物
n. 棕櫚；椰子
adj. 文化的；教養的
adj. 主觀的
Unknown
Unknown
n. 印刷業從業人員
n./v. (繩等的)結
v. 持續
n. 玉米；穀物
n. 紊亂
adv. 此外
n./v. 契約
n. 人行道
n. 男生殖器
n. 煩惱
v. 歡迎
n. 事情；問題
v. 偷
adj./n./pron. 少數
n. 天氣
n. 青蛙
adj. 不法行為
n./v. 休息
n. 傳播
prep./adj./n. 負的
n. 小貓
adj. 客觀的
n. 主廚
adj. 中問的
n./v. 點頭；打瞌睡
Unknown
n. 石油
n. 晚餐
n. 午夜
Unknown
n. v. 清掃；打掃
n. 容器
Unknown
Unknown
Unknown
adj. 輕微的
n. 聖歌
n. 家庭作業
n. 到來
adj. 總統(校長)的
n. 罐；罈；瓶
n./v. 芽；發芽
n. 損害賠償
adj. 敏感的
v. 鼓舞
adj./v. 向前；今後
adj./n. 慣常的
n. 八月
n. 吹風機
v. 使能夠
n./v. 廣播；播放
Unknown
n. 帳戶；描述
Unknown
n. 調查；研究
Unknown
n./v. 佣金；委託
adj./v. 向前；今後
adj. 令人信服的
Unknown
adj. 移動的
n./v. 偶然；機會
adj. 視覺的
n. 監管
n./ v. 輕搖；慢跑
v./n. 再檢查；複習
Unknown
n. 櫥櫃；密室
n. 社區；共同體
n. 書店
adj. 潮溼的
n. 天賦；天才
n. 可能性
n. 行為
n. 選定路線
n. 迎賓員
v. (n)使驚愕
n./v. 轉動
conj.  如果
n./v. 密碼；法典
v. 類得起；承擔得起
v. 撥給
n. 被提名人
n. 優越
n. 命運
adj. 多災多難的
n./v. 電子郵件
n. 議員
n. 蜜月假期
n./v. 飛澱(聲)
n. 木匠
v. 使分心
n. 顧客
n. 地主
n. 軼事
v. 描寫
n./v. 牧羊人；引導
n. 一分錢
n. 咖啡
n. 缺點
adv. 堅固地
pron. 無論是誰
n. 玻璃器皿
Unknown
Unknown
adj./n. 十八
n. 果汁
n. (藥物等的)一劑
n. 真空；空虛(感)
adj. 暫時的
n. 坑；煤礦
v. 歸納
adj. 完全地
n. 揚聲器
n. 例外
n. 櫃檯；籌碼
adj. 必需的
n. 平民；文職人員
n. 自治(權)
n. 邊綠
n. 悲劇
adv. 在旁邊；除…之外
v./n. 抓住；支撐
adv. 不知怎麼地
n. 理解力
n. 統治者；皇帝
adv. 自負地
n./v. 信號
n. 復興
adj. 貪婪的
n. 上下文
Unknown
adj. 過多的
v. 發現
Unknown
n. 哲學家
adj. 巨大的
n. 焦慮
n. 權力
n. 無能
n. 爪子；鉗；贅
n. (陸)龜
n. 缺口；峽谷；隔閡
n. 午餐
Unknown
n. 機車
adj. 突然的；魯莽的
v. 拖
Unknown
n. (大學的) 校園
n. 卡片；紙牌遊戲
n. 共和國
Unknown
n. 慈善團體
Unknown
adj. 美麗的
adj. 擁擠的
v. 使孤立
n. 場景
n. 有義務
adj. 各式各樣的
adj./adv. 一週一次的
v. 承認
n. 論文
n./v. 烤肉
Unknown
n. 宵禁
n./v. 名字
v. 賭博；投機
Unknown
n. 期刊
n. 資訊
n. 人們
v. 使傾覆；推翻
Unknown
adj. 僅僅的
adj. 舒適的
n. 愛情
n. 適應；改編(本)
v. 橫越
Unknown
n./v. 齒輪；搭上齒輪
n. 捕捉者
n. 組織；團體
v. 喚起
n./adj. 較年少者
n. 衛生設備
Unknown
n. 洗髮(精)
n. 協定；公約
v. 使銘記
n. 維納斯女神；金星
Unknown
n. 善良
v. 強姦
Unknown
n. 災難
n. 幾何學
n. 職業
v. 醒來
n. 成見
n./v. 美味；喜愛
n. 所有物
n. 洩漏
n. 國語
n. 電扶梯
n. 飛機副駕駛
n. 警察
Unknown
n. 反抗
Unknown
v. 競爭；比賽
n. 鬼鬼祟祟做事的人
n. 柄；把手
n. 旋轉
n. 商業；交易
n. 誓約；詛咒
n. 頭盔；安全帽
adj. 有影響的
n./v. 鏈條；束縛
v. 予以(打擊等)
n. 生計
v. 爆發；發生
adj. 北方的
v. 減少
adj. 古怪的
adj./n. 紅色的
n. 工程師
Unknown
n. 歌曲
n./v. 地圖
adj. 可取代的
v./n. 展示
n. 樸素
Unknown
adj. 很有可能的；準備前往…的
Unknown
adj. 近視的
n./v. 塔樓；屹立
adv. 驕傲地
Unknown
n. 子彈
adj. 連續的
n. 音樂家
n./v. (=draft)草圖；草擬
n. 協調
adj. 一年一度的
n. 沮喪
n. 大學肄業生
n. 方法；態度
n./v. 頭腦；注意；留意
n. 錄影帶
v. 用力投擲
adj. 緊張的
Unknown
adj. 對…不感興趣
n. 詩人
Unknown
adj./n. 人類的
adj. 幸運的
Unknown
n. 證書
n. 突擊
adj. 現代的
adj. 不耐煩的
n. 菠菜
Unknown
Unknown
n. 十二月
n. 協調
Unknown
n./v. 毀滅
n. 腳掌；腳爪
n. 治療專家
n. 夾克
v. 停止
n. 青少年
n. 環境
n. 飢餓
v. 報…之仇
n. 胡椒粉
n. 拼寫能力
n./v. 保衛；守衛
v. 重新開始
n. 重建(物)
adv. 謙虛地
n. 大砲
adj./adv. 正的
n. 協助
n. 一套衣服
v./n. 希望
adj. 財政的；金融的
v. 介紹；提出
adj. 有尖角的
n. 灌木叢
n./v. 保證；擔保品
n. 心不在焉／抽象概念
v. 慫恿；引誘
n./v. 築巢
n. 黏土；泥土
v. 聽；聽從
n. 童年時期
adj. 令人激動的
Unknown
adj. 確信的
n. 商人
n. 聯盟
n. 批准
v. (使)挨餓
n./v. 榮譽；使…增光
n. 浴室
n. 野兔
n. 西洋棋棋盤
v. 去；從事
v. 修改
v. 使入迷
n. 恐怖
v. 指定；任命
n. 神經
n. 承諾
v. 部分重疊；部分相同
adj. 看得見的
v. 照亮
adj. 有耐心的
n./v. 厄運；注定
adj. 少年的
Unknown
n. 循環；流通；貨幣
n. 藥膏
Unknown
n. 少女
adj. 親密的；熟悉的
adj. 盡責的
n. 葡萄
adv. 同時
adj./adv./pron. 各個
adj./n. 使暗；使不明顯；暗的；不明顯的
n./v. 笑
Unknown
n. 邏輯
n. 長禮服；長抱
n. 創新；創造力
Unknown
n. 多樣性
n. 哩數
n./v. 震驚
adj. 溼的
n./v. 穿；戴
Unknown
adj. 無意義的
v. 來到；甦醒
v. 發怨言
adv./pron./conj.  為什麼
n. [pl. ]腳
n. 山
Unknown
v. 喚起
adj. 光榮的；輝煌的
adj. 聰明的
adj. 感情上的
n. 判決
n. 帳篷
n. 重要性
adj. 比較上的
v. 扣押
Unknown
n. 繼績
adj. 怪物的；恐怖的
n. 沙漠
v. 包括
n. 關閉
n. 藥物治療
n. 操場
Unknown
adj./n. 數字式的
n. 活動；活動力
n. 運送人；航空母艦
n. 結合
n. 獸群
adj. 人造的
n. 耐力
n. 法令；條例
n. 箴言；座右銘
v. 放鬆
n. 啦啦隊隊長
v. (使)終止
Unknown
n./v. 打噴嚏
adj. 做為…後果的
n. 小巷子；小徑
adj. 有同情心的
n. 麥克風
n. 物質
adj. /n易；汽車
Unknown
n. 恃強凌弱者
n. 緊急
n. 競爭者
n. 欣賞；感謝
v. 使滿足
n./v. 吠叫；咆哮
n. 接縫
n. 測量儀表；公尺
Unknown
v. 恢復
adv. 遲疑地
n. 學費
n. 簽字
n. 三點式泳裝
n./v. 分擔；分配
n. 欽佩
n. 繁殖者
n. 新鮮人
n. 危機
n. 背景
Unknown
n. 工程
adj. 學院的；學術的
Unknown
v. 探查
adj. 有暴風雨的
v. 防守；防衛
Unknown
n. 傳教士
n. 眼淚
n. 牆
n. 長襪
adj. 紫外(線)的
Unknown
n. 候選人
n. 剪；切斷
n. 蝗蟲；蚱蜢
n. 雙子座
adj. 興高采烈的
n./v. 喇叭；鼓吹
n. 大湯匙
adj. 有病的
n. 星期三
adj. 自大的
n. 神祕的事物
n./v. 晶片；削
adj. 歡樂的
adj. 專心致志的
n. 便利設施；方便
Unknown
Unknown
n. 山崩
n./v. 電纜；有線電視
v. 關上
Unknown
Unknown
n. 調查者
v. 取消資格
n. 餘波
n. 狀態
adj. 單調的；遲鈍的
adj. 有罪的；內疚的
v. 開(樂方)
Unknown
n. 宗教
n. 哲學
n. 碎片
n. 黑板
adj. 數字的；表示數量
adj. 消瘦的
adj. 個別的
adj. 不切實際的
n. 控制者
Unknown
v. 使驚慌
n./v. 錄音帶
n. 工具
adj. 兇猛的；猛烈的
n. 頑固分子
n. 國家；民族
n. 無決斷力
n. 日曆；行事曆
Unknown
adj. 不可預測的
n. 生長；發育
n. 傷害
n. 啤酒
n. 視力
adv. 非常
adj./v. 滿足的
adj. 可交換的
adj. 一致的
v. 流血
n. 愛好
n. 分類
n. 區域
n. 水桶；桶狀物
n. 傳說
n. 外用藥水；化妝水
Unknown
n. 提名；任命
n. 分隔間
n./v. 衝擊；產生影響
n. 競賽；競爭
Unknown
adv. 舒適地
n. 燕子
n. 宮殿
n./v. 封面；遮蓋
Unknown
adj./n./v. 完全的；總數
n. 描寫；說明?
Unknown
adj. 使人驚訝的
adj. 充足的
adj. 整潔的
adv. 其問；同時
Unknown
n. 公司
n. 私用車道
n. 意義；重要性
n. 行動；作用
adj. 無家可歸的
v. 移居於
n. 質量
n. 小麥
n. 承認
adj./n. 後面的；後半的
n. 中間；媒介
v. 為慶祝
adv. 小心地
adj. 地理學的
v. 證明；展示；遊行
n. 民主
n. 日光
n. 少年
adj./n. 持異議的(人)
v. 悲傷
v. 償還
n. 統治者
adj./v. 狹窄的；限制
n. 外科(手術)
adj. 有教育意義的
v. 隔離開
n. 穀粒；穀物
v. 歸還
Unknown
n. 七月
adj./n. 共和國的；共和主義者
adj. 勇敢的
adj. 安全的
n. 新聞播報員
n. 慷慨
n. 水族館
n./v. 舞蹈
Unknown
adj. 便利的
n. 偵察機
n./v. 擁抱；抱住
adj. 多雪的
adj. 昏昏欲睡的
n. 碗櫃
n./v. 騎；騎乘
n. 陰謀
adj./n. 音樂的
v. 奉獻
n. 行人
n. 討論
n. 應試者
n. 資料
adj. 累積的
n. 身分；同一性
n. 藥；醫學
n. 香味；香水
Unknown
n. 颶風；暴風雨
n. 魔鬼；撒旦
adj. 突起的
Unknown
adj./n. 藍色的
n. 新聞業
v. 提交
adj. 密集的；稠密的
adj. 可愛的
pron. 某人
adj. 更適合的
n. 危害；淘氣
n. 嬰兒；寶貝
v. 打翻
v. 使電腦化
n. 羊
n. 一餐
n. 比薩
adj. 清白的；無知的
n. 百科全書
n. 小狗
n. 投資者
adj. 過剩的
n. 光線
n. 醬料
n. 隊
n./v. 聲音；表達
Unknown
Unknown
Unknown
Unknown
n. 體育館
interj. 乾杯
n./prop. 北京
n. 項目
n. 池塘
n. 福利
n. 小冊子
n. 暴力
Unknown
Unknown
prep./adv. 在遠處
n./v. 貨車；載運
Unknown
Unknown
n. 協會；聯想；合作
n. 母性
n. 一點(撲克牌)；能手
v. 操勞過度
n/v. 速度；急行
Unknown
n. 肌肉
n. 控訴
Unknown
v. 做不正當的事
adj. 徹底的
n. 計算
n. 生產力
adj. 構成整體所需要的
v./n. 摸；感覺
v. 耕(地)
adj. 有益的
n/adv. 那裡
adj. 偶爾的
Unknown
n. 竹子
adj. 壯麗的
n. 人力
n. 圖解；說明?
n. (美、加等國)元
adj. 省的
n./v. 衝突
n. 收藏者
adj. 可怕的
n. 成員
adj./v.  輪流的
pron. 誰
n. 袋子
n. 金屬
adj. 相似的
n. 代名詞
n. 錢包
ad v. 慷慨地
n./v. 群；聚集
n. 攻擊
n. 原野；欄位
n. 遠足
n./v. 吹；吹氣；打擊
n. 主管；導演
v./adj. 稀釋(的)
n. 短缺
v. 溝通
n. 慣例
n. 貨物
v. 犯錯
v. 預見
n./adv. 這裡；在這裡
n. 樹幹
n. 隆隆聲；景氣
adv. 完全；非常
adv. 幽默地
adj. 傳統的
n./v. 數字；給…號碼
n. 實驗室
n./v. 吸引；呼籲
n./v. 運動；鍛鍊
n. 鄉村
adj. 有害的
Unknown
n. 演講者
n. 革新
n. 書架；書櫥
adj./n. 盡頭的；極限
Unknown
n. 直昇機
n. 枕頭
adj./adv. 位於兩地中途的
v. 表現；行為舉止
n. 發電機
Unknown
adj. 各種各樣的
n. 物體；目標
adj./adv./n. 二分之一的
Unknown
n./v. 氣味；聞
n. 夥伴
Unknown
Unknown
n. 電；電流
v. 使給予活力
Unknown
adj. 宇宙的
adj. 高興的
Unknown
n. 板
n. 聯結；契約；債券
n. 時鐘
adj. 粗心大意的
v. 圍住；把(公文等)封入
n. (重量單位)克
